src/main/java/InputProcess.kt

```diff
@@ -0,0 +1,278 @@
+                return false
+            }
+        }
+
+        if (inputString.isEmpty())
+            return false
+        return true
+    }
+
+    private fun outputArrayInspector(args: MutableList<String>): Boolean {
+        var isFormatSet = false
+        var isStringSet = false
+
+        if (args.isEmpty()) {
+            println("No arguments for output. Default settings will used.")
+            return true
+        }
+
+        args.forEachIndexed { _, value ->
+            when (value) {
+                "-hex" -> {
+                    if (!isFormatSet) {
+                        isFormatSet = true
+                        outputFormat = HEX_FORMAT
+                        Logger.write("Output format hex defined")
+                    } else {
+                        println("Wrong output part of args: $value")
+                        return false
+                    }
+                }
+                "-base64" -> {
+                    if (!isFormatSet) {
+                        isFormatSet = true
+                        outputFormat = BASE64_FORMAT
+                        Logger.write("Output format base64 defined")
+                    } else {
+                        println("Wrong output part of args: $value")
+                        return false
+                    }
+                }
+                "-dec" -> {
+                    if (!isFormatSet) {
+                        isFormatSet = true
+                        outputFormat = DEC_FORMAT
+                        Logger.write("Output format dec defined")
+                    } else {
+                        println("Wrong output part of args: $value")
+                        return false
+                    }
+                }
+                "-bin" -> {
+                    if (!isFormatSet) {
+                        isFormatSet = true
+                        outputFormat = BINARY_FORMAT
+                        Logger.write("Output format bin defined")
+                    } else {
+                        println("Wrong output part of args: $value")
+                        return false
+                    }
+                }
+                else -> {
+                    if (value[0] != '-' && !isStringSet) {
+                        isStringSet = true
+                        outputFilePath = value
+                        Logger.write("Output filepath string defined")
+                    } else if (value != "-o") {
+                        println("Wrong output part of args: $value")
+                        return false
+                    }
+                }
+            }
+        }
+
+        if (!isFormatSet) {
+            if (isToEncode)
+                outputFormat = BASE64_FORMAT
+            else if (isToDecode)
+                outputFormat = DEC_FORMAT
+        }
+        if (isStringSet) {
+            isPrintIntoFile = true
+        }
+        return true
+    }
+
+
```

Comment:
Inconsistent separation of methods: 2 blank lines vs 1 blank line between all other methods.